netutils/ping/icmpv6_ping.c

```diff
@@ -261,7 +280,7 @@ void icmp6_ping(FAR const struct ping6_info_s *info)
           recvfd.events   = POLLIN;
           recvfd.revents  = 0;
 
-          ret = poll(&recvfd, 1, info->timeout - elapsed);
+          ret = poll(&recvfd, 1, info->timeout - elapsed / USEC_PER_MSEC);
```

Comment:
Don't we need the same change in `netutils/ping/icmp_ping.c`?

Reply:
It has been modified